uaclient/contract.py

```diff
@@ -171,10 +171,13 @@ def request_machine_token_refresh(
             machine_id = util.get_machine_id(self.cfg.data_dir)
         headers = self.headers()
         headers.update({"Authorization": "Bearer {}".format(machine_token)})
-        url = API_V1_TMPL_CONTEXT_MACHINE_TOKEN_REFRESH.format(
+        url = API_V1_TMPL_CONTEXT_MACHINE_TOKEN_UPDATE.format(
             contract=contract_id, machine=machine_id
         )
-        response, headers = self.request_url(url, headers=headers)
+        platform = util.get_platform_info()
```

Comment:
This looks like a copy/paste of the code two hunks above here; should we refactor?

Reply:
sure we can add a _get_platform_header private method to surface those keys

Reply:
Make that _get_platform_data instead. done.